backend/app/views/spree/admin/option_types/edit.html.erb

```diff
@@ -16,11 +16,12 @@
 <%= render :partial => 'spree/shared/error_messages', :locals => { :target => @option_type } %>
 
 <%= form_for [:admin, @option_type] do |f| %>
-  <fieldset>
-    <legend align="center"><%= Spree::OptionValue.model_name.human(count: :other) %></legend>
-
+  <fieldset class="no-border-bottom">
+    <legend><%= Spree::OptionType.model_name.human %></legend>
     <%= render :partial => 'form', :locals => { :f => f } %>
-
+  </fieldset>
+  <fieldset class="no-border-bottom">
```

Comment:
Sorry, I've been playing with these fieldsets all day long - and I think this one should actually have a bottom border as it's the last fieldset on the page. Would you mind amending this again?